src/releng_docs/deploy/regular.rst

```diff
@@ -1,31 +1,73 @@
-.. _deploy-weekly-releases:
+.. _deploy-regular:
 
-Regular weekly releases
-=======================
+Regular deployment
+==================
 
-Push to production happen in weekly batches. Once a week (usually on Thursday)
-a release happens.
+Regular production deployment of all projects happens once every two weeks.
 
-.. _deploy-release-managers:
+.. _deploy-managers:
 
-Current administrators that perform this weekly release are:
+Administrators that perform this regular release are:
 
 - `Rok Garbas`_
 - `Bastien Abadie`_
 - `Rail Aliiev`_
+- `Jan Keromnes`_
 
-Protocal that we follow is:
+Release schedule is published in `Release Services calendar`_. Protocol we follow is:
 
+TODO: this should happen on Wed morning, point to the calendar
 
-#. Prior to release a push to ``staging`` branch must happen. This will
-   trigger a deploy of all projects to staging environments.
+#. Push to staging channel.
+
+   A day before pushing to production, on Wednesday morning, we push to
+   projects to **staging channel**.
+
+   To trigger automatic deployment to staging channel you need to force push
+   from ``master`` to ``staging`` branch.
 
    .. code-block:: console
 
         $ git clone git@github.com:mozilla/release-services.git
         $ cd release-services
         $ git push -f origin origin/master:staging
 
+   Once pushed to staging branch a deployment will start via taskcluster github
+   integration. You can find a link to taskcluster deployment group at the
+   `page listing commits of staging branch`_.
+
+   .. image:: page_listing_commits_of_staging_branch.png
+
+   To make everybody aware that this is happening announce staging deployment on
+   `#release-services` IRC channel.::
+
+      I'm pushed commit `<SHORT_SHA_OF_COMMIT>' to staging branch. Until
```

Comment:
Nit: s/I'm/I've/.

Nit: Is it really necessary to copy/paste the commit SHA? I'd just say "I've pushed to staging branch" (people can easily find the SHA if they really need it).